docs/arch/adr-04-merging-mainstream-to-specialist-topics.md

```diff
@@ -0,0 +1,27 @@
+# Merging of Mainstream Topics into Specialist Topics
+
+## Status
+
+Proposed
+
+## Context
+
+GOV.UK currently maintains 2 separate topic trees that represent a navigational structure for users. Mainstream Browse is the oldest and considered legacy. Specialist Topics are newer but need tidying up and do not power breadcrumbs.
+
+Previous attempts to merge the two stalled (since 2015).
+
+The newest tree, Taxonomy, does not affect navigation and is treated as a purely metadata type categorisation tree and is ignored by this work.
+
+## Decision
+
+We initially planned to map each Mainstream Browse Topic to an equivalent within the Specialist Topics tree. This would have been complex and difficult to even begin the merging of the two trees. 
+
+But deeper discussion brought us to realise that there are no overlaps, allowing us to naively merge the trees and tidy up the data in one place. 
+
```

Comment:
I think we could expand this, I don't really understand what no overlaps means.

Reply:
Does it look better now?